ntbs-service-tests/UnitTests/Patients/IndexPageTest.cs

```diff
@@ -24,9 +24,11 @@ public IndexPageTest()
         public async Task OnGetAsync_PopulatesPageModel_WithPatients()
         {
             // Arrange
-            mockService.Setup(service => service.GetRecentNotificationsAsync())
+            // TB service is currently hard coded on Index.cshtml.cs while user logins are missing so this needs to match
+            var mockTBServices = new List<string> {"Ashford Hospital"};

```

Comment:
I'm not sure if I can arrange the 'hard coding' on index.cshtml.cs to mean this isn't necessary to match it but this makes the test pass 🤷‍♂ When user log in is brought in this will need to be fixed